Sources/ParseSwift/Objects Protocols/ObjectType.swift

```diff
@@ -133,11 +141,20 @@ func getJSONEncoder() -> JSONEncoder {
 private let forbiddenKeys = ["createdAt", "updatedAt", "objectId", "className"]
 
 func getParseEncoder() -> ParseEncoder {
+    return getParseEncoder(skipKeys: true)
+}
+
+func getParseEncoderWithoutSkippingKeys() -> ParseEncoder {
```

Comment:
Is this function necessary? Can we just call `getParseEncoder` directly and give it the `skipKeys` param?

Reply:
In it's current setup, you can call `getParseEncoder` with the `skipKeys` parameter. I added `getEncoderWithoutSkippingKeys` as an extension of `ObjectType`, `getParseEncoder` isn't part of that extension. These are things I didn't want to move around because it can easily create merge conflicts for your PR, but it's possible depending on the changes in your PR that we can do something like this after it's merged into yours